app/assets/stylesheets/layout_components/buttons/buttons.scss

```diff
@@ -68,6 +68,20 @@
     }
   }
 
+  &--white {
```

Comment:
Do you have an example of where this is used @sineade?

Reply:
* spoke to dave about this one in person